### ISSUE TITLE
Is it possible to use via the e-books from melonbooks?

### ISSUE THREAD

**Issue by enforcing** [apprenticeharper/DeDRM_tools#1065]
That the format is `*.melon` or `*.pdf.ebg`. What is the function of `.ebg`? Can we remove it without causing damage to the pdf file?

**Comment by ElleKayEm** [comment]
No, these tools do not work for that DRM.  No idea about .ebg.  Sounds like some sort of wrapper.  No idea how you'd extract the pdf.

**Comment by enforcing** [comment]
![圖片](https://user-images.githubusercontent.com/63993990/79761388-f25e1e00-8353-11ea-8cab-4ccc25339b90.png)

**Comment by enforcing** [comment]
Could I submit this document for testing?

**Comment by ElleKayEm** [comment]
It's doubtful that Apprentice Harper would be interested.  He barely has time these days to maintain the current tools.  But his email is apprenticeharper@gmail.com if you want to ask.
